integration/containerd/cri/integration-tests.sh

```diff
@@ -336,7 +336,9 @@ main() {
 	pushd "${GOPATH}/src/${cri_repo}"
 
 	git reset HEAD
-	git checkout release/1.3
+	git checkout master
```

Comment:
I don't think we should do that, @jongwu.

There are a whole bunch of things that would need to be changed that would be way too intrusive for a backport.

Reply:
hello @fidencio -, I'm trying to fix the arm ci issue on stable-2.1, so I come up with that changing the branch for containerd. but it doesn't work. I will close this PR, but I want some idea from you about how to save the arm ci. see http://jenkins.katacontainers.io/job/kata-containers-2.0-ubuntu-ARM-PR/2110/consoleFull